src/webgl/p5.Camera.js

```diff
@@ -1746,7 +1746,8 @@ p5.Camera.prototype._orbit = function(dTheta, dPhi, dRadius) {
 
   // calculate updated camera angle
   // Find the angle between the "up" and the "front", add dPhi to that.
-  const camPhi = front.angleBetween(up) + dPhi;
+  const camPhi =
```

Comment:
Makes sense, can we add a comment explaining what this does differently from `angleBetween` for context for future readers?

Reply:
Should I mention the possibility of getting negative numbers?
"angleBetween() can return negative numbers. Since this specification is subject to change due to version updates, it cannot be adopted, so here we calculate using a method that directly obtains the absolute value."
Is this better?

Reply:
I think mentioning negative numbers is good, since that's main motivation and the bit that would cause a bug (and using a separate implementation instead of abs() just makes it a bit more efficient and less likely to break in the future.) Your text looks good!